content/blog/2024-07-31-valkey-8-0-0-rc1.md

```diff
@@ -0,0 +1,126 @@
+  will automatically inherit the correct slot migration state, ensuring immediate
+  consistency.
+
+## Replication
+
+Valkey 8.0 introduces a dual-channel replication scheme, allowing the RDB and
+the replica backlog to be transferred simultaneously, accelerating synchronization.
+
+- **Reduced Memory Load**: By streaming replication data to the replica during
+  the full sync, the primary node experiences significantly less memory pressure.
+  The replica now manages the Client Output Buffer (COB) tracking, reducing the
+  likelihood of COB overruns and enabling larger COB sizes on the replica side.
+- **Lighter Main Process Load**: A dedicated connection for RDB transfer frees
+  the primary's main process from handling this data, allowing it to focus on
+  client queries and improving overall responsiveness.
+
+Performance tests show improvements in write latency during sync, and in scenarios
+with heavy read commands, the sync time can be cut by up to 50%. This translates
+to a more responsive system, even during synchronization.
+
+## Observability
+
+Valkey 8.0 introduces a comprehensive per-slot metrics infrastructure, providing
+detailed visibility into the performance and resource usage of individual slots.
+This granular data helps inform decisions about resource allocation, load
+balancing, and performance optimization.
+
+- **CPU Usage**: Tracks CPU time consumed by operations on each slot, identifying
+  areas of high utilization and potential bottlenecks.
+- **Network Input/Output Bytes**: Monitors data transmission and reception by
+  each slot, offering insights into network load and bandwidth utilization.
+- **Minimal Overhead**: Initial benchmarks show that enabling detailed metrics
+  incurs a negligible overhead of approximately 0.7% in QPS.
+
+## Efficiency
+
+Valkey 8.0 introduces key embedding in the main dictionary, eliminating separate
+key pointers and significantly reducing memory overhead. This results in a 9-10%
+reduction in overall memory usage for scenarios with 16-byte keys and 8 or 16-byte
+values, along with slight performance improvements.
+
+## Additional Highlights
+
+- **Dual IPv4 and IPv6 Stack Support**: Seamlessly operate in mixed IP environments
+  for enhanced compatibility and flexibility.
+- **Improved Pub/Sub Efficiency**: Lightweight cluster messages streamline
+  communication and reduce overhead for faster, more efficient Pub/Sub operations.
+- **Valkey Over RDMA (Experimental)**: Unlock significant performance improvements
+  with direct memory access between clients and Valkey servers, delivering up to
+  275% increase in throughput and over 50% latency reduction in certain scenarios.
+- **Numerous Smaller Performance/Reliability Enhancements**: Many under-the-hood
+  improvements ensure a smoother, more stable experience across the board.
+
+## Conclusion
+
+Valkey 8.0 is a comprehensive update that offers improved performance, reliability,
```

Comment:
```suggestion
Valkey 8.0 is a significant update that offers improved performance, reliability,
```
Or major. Just don't say comprehensive, since that is vague.